dspace-api/src/main/java/org/dspace/content/authority/MetadataAuthorityServiceImpl.java

```diff
@@ -83,6 +83,11 @@ protected MetadataAuthorityServiceImpl() {
 
     }
 
+    @Override
+    public void reset() {
```

Comment:
I feel like we should also clear the other map in this method. A reset method within a service should reset each variable that's caching information, this way there can't be any confusion later down the line as to why the other one isn't getting reset when this method is called. I'm referring to the `controlled` variable here